libs/components/theme/src/lib/theming/fixtures/theme-component-class-test.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+import { SkyTheme } from '../theme';
+import { SkyThemeComponentClassDirective } from '../theme-component-class.directive';
+import { SkyThemeMode } from '../theme-mode';
+import { SkyThemeSettings } from '../theme-settings';
+
+@Component({
+  selector: 'app-theme-component-class-test',
+  templateUrl: './theme-component-class-test.component.html',
+  standalone: true,
+  hostDirectives: [SkyThemeComponentClassDirective],
+})
+export class SkyThemeComponentClassTestComponent {
+  public themeSettings: SkyThemeSettings | undefined;
+
+  constructor() {
+    this.useDefaultTheme();
+  }
+
+  public useDefaultTheme(): void {
```

Comment:
I don't see where this method or the method below it is called or what it would do when it is called since it never interacts with the theme service. Is this left over from a previous iteration?